src/java/com/sarality/app/view/dialog/BaseDialogFragment.java

```diff
@@ -27,8 +17,8 @@
   @Override
   public Dialog onCreateDialog(Bundle savedInstanceState) {
     AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
-    getActionManager().setup(builder);
     configure(builder);
+    getActionManager().setup(builder);
```

Comment:
the buttons were getting configured in configure(builder) but the setup already called which means that the extension class was not getting a chance to set the actions on the buttons.